generator.yaml

```diff
@@ -28,8 +28,14 @@ resources:
           input_fields:
             PolicyName: Name
     hooks:
+      sdk_read_one_post_set_output:
+        template_path: hooks/policy/sdk_read_one_post_set_output.go.tpl
       sdk_create_post_set_output:
         template_path: hooks/policy/sdk_create_post_set_output.go.tpl
+      sdk_update_post_set_output:
+        template_path: hooks/policy/sdk_update_post_set_output.go.tpl
```

Comment:
you can remove this because when you use a `custom_method_name`, the `sdk_update_post_set_output` generic hook is not evaluated.